src/hooks/identity-hooks.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery, type UseQueryOptions } from "@tanstack/react-query";
+import { type ManagedIdentity } from "@web5/agent";
+import { IdentityAgentManager } from "@/features/identity/IdentityAgentManager";
+import { type CustomizableUseQueryOptions } from "@/hooks/types";
+
+export const useIdentityList = (
+  options: CustomizableUseQueryOptions<ManagedIdentity[]> = {}
```

Comment:
no need for this param as far as i can tell

Reply:
Currently, no. I can remove if we'd like. 

I left it here because these `CustomizableUseQueryOptions` do allow the caller to provide their own `UseQueryOptions`, except for the `queryFn` and `queryKey`. This would include things like retry intervals, length of time results are in the cache, initial data, etc.